civiform_config.example.sh

```diff
@@ -184,13 +190,27 @@ export CUSTOM_HOSTNAME=""
 export APP_PREFIX="my-deploy" # max 19 chars, only numbers, letters, dashes, and underscores
 
 # REQUIRED
-# Which auth provider to use for applicants to login.
-# If set to a non-disabled value, you must configure the respective auth parameters
-export CIVIFORM_APPLICANT_IDP="generic-oidc"
+# ARN of the SSL certificate that will be used to handle HTTPS traffic. The certiciate
+# should be created and validated before the deployment is done. Certificate can be created
+# in AWS web console: https://console.aws.amazon.com/acm/home#/certificates/list
+export SSL_CERTIFICATE_ARN=""
```

Comment:
this is okay for now, but it would be great if we could create and validate this automatically.

Reply:
That's what I'm not quite certain about. I believe we can create and validate automatically well when domain is fully controlled by AWS (via route53). But if domain DNS records controlled elsewhere, then certificate will have to be created manually and DNS validation will require owner to go the DNS settings in at the hosting provider and update it there. 
Do we expect civic entities all use AWS to handle domain DNS settings?